types/known/anypb/any.pb.go

```diff
@@ -58,7 +58,7 @@
 // of that message can be passed to the UnmarshalTo method:
 //
 //	m := new(foopb.MyMessage)
-//	if err := any.UnmarshalTo(m); err != nil {
+//	if err := anypb.UnmarshalTo(m); err != nil {
```

Comment:
This call here is to a receiver method call on `anypb.Any`: [`UnmarshalTo`](https://github.com/protocolbuffers/protobuf-go/blob/master/types/known/anypb/any.pb.go#L357-L362)

The signature to the top-level function in `anypb.UnmarshalTo` is entirely different, and does not match this call at all:
```
func UnmarshalTo(src *Any, dst proto.Message, opts proto.UnmarshalOptions) error
```